components/kubeflow/kfserving/component.yaml

```diff
@@ -12,7 +12,8 @@ inputs:
   - {name: Canary Custom Model Spec,        type: String, default: '{}', description: 'Custom runtime canary custom model container spec.'}
   - {name: Autoscaling Target,              type: String, default: '0',        description: 'Autoscaling Target Number'}
   - {name: KFServing Endpoint,              type: String, default: '', description: 'KFServing remote deployer API endpoint'}
-  - {name: Service Account,                type: String, default: '', description: 'Model Service Account'}
+  - {name: Service Account,                 type: String, default: '', description: 'Model Service Account'}
+  - {name: Istio Sidecar,                   type: String, default: 'True',     description: 'Whether to enable istio sidecar injection'}
```

Comment:
Its a boolean, should the name signify that?

Reply:
Sorry for the late reply, I addressed this with my latest commit.